### PR TITLE
Add SystemHeat configs to RO

### PR DESCRIPTION
**Pull request by Capkirk123**
Add SystemHeat configs to RO nuclear reactors. These patches are designed so that they will work the same regardless of whether SystemHeat's own patches run

Also modify stock radiators to function purely as cryocoolers, and HeatControl Radiators to function as passive radiators.

### CODE REVIEW

File: GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/HeatControlMicrochannelRadiators.cfg [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+		
+		//Since this radiator is passive, it can only cool things hotter than it is
+		@overcoolFactor = 1
+
+	!MODULE[ModuleB9PartSwitch] {}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = sizeSwitch
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/HeatControlMicrochannelRadiators.cfg:197]: Are we certain there is exactly 1 ModuleB9PartSwitch on these parts, or should this delete using `:HAS`?

File: GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/SquadRadiators.cfg [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+@PART[radPanelEdge]:AFTER[SystemHeat]
+{
+	@title = Cryocooler Radiator Panel (Fin)
+	@description = Cryocooler, for keeping cryogenic propellants cool. Install HeatControl for high temperature radiators.
+	
+	@MODULE[ModuleSystemHeatRadiator]
+	{
+		@parentCoolingOnly = false
+	}
+}
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/SquadRadiators.cfg:13]: DRY this file?
```
@PART[radPanelEdge]:AFTER[SystemHeat] { @title = <unique> }
@PART[a bunch of parts]:AFTER[SystemHeat]
{
	@description = Cryocooler, for keeping cryogenic propellants cool. Install HeatControl for high temperature radiators.
	@MODULE[ModuleSystemHeatRadiator] {@parentCoolingOnly = false}
}
```

File: GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/HeatControlMicrochannelRadiators.cfg (hunk 1 above)
Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/HeatControlMicrochannelRadiators.cfg:197]: Nertea himself does the exact same thing in the SystemHeat patches for HeatControl. There is only one MB9PS present.

**Comment by Capkirk123** [comment]
Patches cleaned up.

**Review by DRVeyl — APPROVED** [review]
Other than extra repetitions of the patches removing MODULEs, and maybe it's a little suspect setting fields typically set in FOR[RealismOverhaul] (part titles/manufacturer/desc/mass/scale, all of which are at risk of 3P mods overwriting them after our typical pass) in our late zzzRealismOverhaul, this looks fine.  Would appreciate a couple others looking it over and testing.

**Pull request merged**